scene/2d/gpu_particles_2d.cpp

```diff
@@ -874,7 +874,7 @@ GPUParticles2D::GPUParticles2D() {
 	set_randomness_ratio(0);
 	set_visibility_rect(Rect2(Vector2(-100, -100), Vector2(200, 200)));
 	set_use_local_coordinates(false);
-	set_draw_order(DRAW_ORDER_LIFETIME);
+	set_draw_order(DRAW_ORDER_INDEX);
```

Comment:
This breaks compatibility

Reply:
Okay, reverted that change.

Reply:
This could be done in a way that only affects newly created nodes in the editor, so that it doesn't break compatibility. We don't have a standard way to achieve this yet though.

Reply:
I appreciate that suggestion and it could be something I could come back to at some point. At the moment I am trying to help with small basic tasks to get familiar with the code base so that is probably a bit out of the scope for me for now.